src/ComplexMethod.java

```diff
@@ -22,6 +22,25 @@ public String intToEnglishValue(int i) {
         if (i > 6) {
             throw new NotImplementedException();
         }
+		if (i > 7) {
+            return 7;
+        }
+		if (i > 8) {
+            return 8;
+        }
+		if (i > 9) {
+            return 9;
+        }
+		if (i > 12) {
+            return 11;
+        }
+		if (i > 10) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 10 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS109)